### PR TITLE
added image component

### PR DESCRIPTION
**Pull request by Samarth-5109**

### CODE REVIEW

File: components/images/images.css [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+
+
+.btn-wrapper{
+    margin-top: 3rem;
+}
+/* FOR IMAGES */
+
+.responsive-image{
+    max-width: 60%;
```
Comment (SukanyaSen530) [inline components/images/images.css:12]: should it always take 60% of the parent container?

File: components/images/images.html [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+            </div>
+
+            <div class="type-wrapper btn-wrapper">
+                <div class="component-type">
+                    <span class="type-title"> Round Image</span>
+                </div> 
+                <!-- <div class="about-type">
+                    <p class="about">There are multiple sizes for square avatars as well. </p> 
+                </div> -->
```
Comment (tewarimohit) [inline components/images/images.html:79]: Remove unnecessary comments...

**Review by tewarimohit — APPROVED** [review]
Just go through suggested changes... otherwise good to merge...

File: components/images/images.css (hunk 1 above)
Comment (Samarth-5109) [reply components/images/images.css:12]: corrected, I made it relative to a diiferent container now  made some changes. Thanks

**Review by SukanyaSen530 — APPROVED** [review]

**Pull request merged**